features/time.yml

```
@@ -0,0 +1,12 @@
+name: <time>
+description: The `<time>` HTML element represents a specific period in time. It may include the `datetime` attribute to translate dates into machine-readable format.
```

I like this! What do you think of adding some examples for the "period in time", like this?

```suggestion
description: The `<time>` HTML element represents a time, such as a calendar date, clock time, or duration. It may include the `datetime` attribute to translate dates into machine-readable format.
```